node/relay_request.js

```diff
@@ -51,11 +52,33 @@ RelayRequest.prototype.createOutRequest = function createOutRequest(host) {
         return;
     }
 
+    self.peer = self.channel.peers.choosePeer(null, {
+        host: host
+    });
+    if (!self.peer) {
+        self.onError(errors.NoPeerAvailable());
+        return;
+    }
```

Comment:
There are no retries because we call `peer.request()` instead of `channel.request()`

All retries are in the logical request class which is only driven from the `channel`